cdxev/__main__.py

```diff
@@ -227,7 +227,14 @@ def create_merge_parser(
 def create_merge_vex_parser(
     subparsers: argparse._SubParsersAction,
 ) -> argparse.ArgumentParser:
-    parser = subparsers.add_parser("merge-vex", help="Merges a VEX file into an SBOM.")
+    parser = subparsers.add_parser(
+        "merge-vex",
+        help=(
+            "[Deprecated] - DO NOT USE. This will be removed in a future version."
```

Comment:
I'd remove the `DO NOT USE`. First of all, that's implied by _deprecated_ and secondly, we don't have a replacement, yet. 

Telling people not to use the command without providing them an alternative is a bit weird 😉 

Same goes for `available_commands.md`